src/ai/ai.cpp

```diff
@@ -346,7 +348,15 @@ void ShipAI::runOrders()
                 flyTowards(owner->getOrderTargetLocation());
             }
         }else{
```

Comment:
Will unarmed ships with roaming orders within 15U of a friendly station always broadcast that they're out of ammo and go into Repair mode? If so, this statement might need a narrower condition.